src/main/java/io/codelex/arrays/practice/My_TicTacToe.java

```diff
@@ -0,0 +1,174 @@
+                System.out.println("You Win!");
+                break;
+            }
+            if (table[0][0].equals(playerX) && table[1][0].equals(playerX) && table[2][0].equals(playerX)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][1].equals(playerX) && table[1][1].equals(playerX) && table[2][1].equals(playerX)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][2].equals(playerX) && table[1][2].equals(playerX) && table[2][2].equals(playerX)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][0].equals(playerX) && table[1][1].equals(playerX) && table[2][2].equals(playerX)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][2].equals(playerX) && table[1][1].equals(playerX) && table[2][0].equals(playerX)) {
+                System.out.println("You Win!");
+                break;
+            }
+
+            notEquals();
+
+            player2();
+
+            if (table[0][0].equals(playerO) && table[0][1].equals(playerO) && table[0][2].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[1][0].equals(playerO) && table[1][1].equals(playerO) && table[1][2].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[2][0].equals(playerO) && table[2][1].equals(playerO) && table[2][2].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            }
+            if (table[0][0].equals(playerO) && table[1][0].equals(playerO) && table[2][0].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][1].equals(playerO) && table[1][1].equals(playerO) && table[2][1].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][2].equals(playerO) && table[1][2].equals(playerO) && table[2][2].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][0].equals(playerO) && table[1][1].equals(playerO) && table[2][2].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            } else if (table[0][2].equals(playerO) && table[1][1].equals(playerO) && table[2][0].equals(playerO)) {
+                System.out.println("You Win!");
+                break;
+            }
+        }
+    }
+
+    public static void emptyBoard() {
+
+        for (int i = 0; i < 3; i++) {
+            for (int j = 0; j < 3; j++) {
+                table[i][j] = " _ ";
+            }
+        }
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
Šī laukuma izvadīšana arī izskatās, ka atkārtojas vismaz 3x, iznes to atsevišķā metodē un vienkārši izsauc, kur vajag.